src/libraries/System.DirectoryServices.Protocols/tests/DirectoryServicesProtocolsTests.cs

```diff
@@ -774,13 +786,25 @@ private SearchResultEntry SearchUser(LdapConnection connection, string rootDn, s
             return null;
         }
 
+        private LdapConnection GetConnection(string server)
+        {
+            LdapDirectoryIdentifier directoryIdentifier = new LdapDirectoryIdentifier(server, true, false);
```

Comment:
Nit, when passing true or false to a method it's nice to name the parameters so it's way to see what they are